configuration-sample/ods-core.env.sample

```diff
@@ -46,6 +46,10 @@ NEXUS_PASSWORD_B64=changeme
 # Nexus (basic) authentication username:password
 NEXUS_AUTH=developer:changeme
 
+# Nexus request/limits.
+# You can configure them here: https://github.com/opendevstack/ods-core/blob/master/nexus/ocp-config/bc.yml#L14-L21
+# For instance,
+# NEXUS_BUILD_CPU_REQUEST=200m
```

Comment:
For me this sounds like people should edit https://github.com/opendevstack/ods-core/blob/master/nexus/ocp-config/bc.yml#L14-L21. But that's not really what we want ... I think we should tell the user that all params in that template can be overwritten by specifying the param in this file here ...

Reply:
That's right - it gives a false impression that they should edit the params, well spotted!